udata_front/theme/gouvfr/assets/js/api/dataservices.ts

```diff
@@ -1,6 +1,22 @@
-import { api, type Dataservice } from "@datagouv/components/ts";
+import { api, type Dataservice, type Dataset } from "@datagouv/components/ts";
+import type { PaginatedArray } from "./types";
+import { getLocalizedUrl } from "../i18n";
 
 export async function getDataservice(id: string) {
   const res = await api.get<Dataservice>(`dataservices/${id}/`);
   return res.data;
 }
+
+export async function getOrganizationDataservices(oid: string, page: number, pageSize: number, sortDirection: string) {
+  const resp = await api.get<PaginatedArray<Dataservice>>(getLocalizedUrl(`dataservices/`), {
```

Comment:
I think private or deleted dataservices aren't fetched on this endpoint yet.
I think we need to apply the same logic on the backend as it has been done for reuses. Ticket created: https://github.com/datagouv/data.gouv.fr/issues/1549

Reply:
🙏